apps/rush-lib/src/api/RushConfiguration.ts

```diff
@@ -775,6 +777,23 @@ export class RushConfiguration {
     return this._repositoryUrl;
   }
 
+  /**
+    * Odd-numbered major versions of Node.js are experimental.  Even-numbered releases
+    * spend six months in a stabilization period before the first Long Term Support (LTS) version.
+    * For example, 8.9.0 was the first LTS version of Node.js 8.  Pre-LTS versions are not recommended
+    * for production usage because they frequently have bugs.  They may cause Rush itself
+    * to malfunction.
+    *
+    * Rush normally prints a warning if it detects a pre-LTS Node.js version.  If you are testing
+    * pre-LTS versions in preparation for supporting the first LTS version, you can use this setting
+    * to disable Rush's warning.
+    *
+    * @beta
```

Comment:
I'm not sure this needs to be marked as `@beta`.  The rush.json schema change will impact compatibility.